schedule/schedule-appengine/src/main/java/com/jasify/schedule/appengine/spi/ActivityEndpoint.java

```diff
@@ -64,12 +64,18 @@
                 packagePath = ""))
 public class ActivityEndpoint {
 
+    private final ActivityDao activityDao = new ActivityDao();
+    private final ActivityPackageDao activityPackageDao = new ActivityPackageDao();
+    private final ActivityPackageActivityDao activityPackageActivityDao = new ActivityPackageActivityDao();
+    private final ActivityTypeDao activityTypeDao = new ActivityTypeDao();
+    private final OrganizationDao organizationDao = new OrganizationDao();
+
     @ApiMethod(name = "activityTypes.query", path = "activity-types", httpMethod = ApiMethod.HttpMethod.GET)
     public List<ActivityType> getActivityTypes(User caller, @Named("organizationId") Key organizationId) throws NotFoundException {
         checkFound(organizationId);
         try {
-            Organization organization = OrganizationServiceFactory.getOrganizationService().getOrganization(organizationId);
-            return ActivityServiceFactory.getActivityService().getActivityTypes(organization);
+            Organization organization = organizationDao.get(organizationId);
+            return activityTypeDao.getBy(organization);
         } catch (EntityNotFoundException e) {
```

Comment:
This I don't like.  Why do we need to get the organization if all we need in the getBy query is the organizationId?

I'm all for a `activityTypeDao.getByOrganizationId(organizationId);`, you dislike it?